build-aux/Jenkinsfile.full

```diff
@@ -703,6 +703,7 @@ pipeline {
           sh '''
             cp js/centos-7/*rpm pkgs/centos7
             cp js/centos-8/*rpm pkgs/centos8
+            cd pkgs
             cd ../centos7 && createrepo --database .
```

Comment:
this is not workin since now you are here in the tree:
```
parent
   ├─pkgs <-- you will be here
      ├─ centos7
      ├─ centos8
```
If you change the directory to the parent of `pkgs`, there is no `centos7` or `centos8` in it.

If you do a `cd/centos7` you can swich from there to `centos8` with `../centos8`.

Reply:
Done in f9846eb